### DISCUSSION TITLE
Module Path Alias

### DISCUSSION THREAD

**Discussion by thelicato** [tinacms/tinacms#4215] [Ideas]
Hello everyone, I started a new project using the Tailwind starter. I absolutely hate relative imports so I configured the ``tsconfig.json`` as I usually do by adding the following:
```
    "paths": {
      "@/*": ["./*"]
    }
```

Even if the app runs I get multiple The following dependencies are imported but could not be resolved: errors. Is there any way to make the absolute imports work smoothely with Tina? If not I think it is a needed feature!

**Comment by kldavis4** [comment]
<img width="586" alt="Screen Shot 2023-09-22 at 09 36 45" src="https://github.com/tinacms/tinacms/assets/370955/019e9b4d-d5a2-4356-8fb5-b4085c80bf9c">
